src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -1271,7 +1271,7 @@ public static partial class JsonTypeInfoResolver
     {
         public static System.Text.Json.Serialization.Metadata.IJsonTypeInfoResolver Combine(params System.Text.Json.Serialization.Metadata.IJsonTypeInfoResolver?[] resolvers) { throw null; }
     }
-    public abstract partial class JsonTypeInfo<T> : System.Text.Json.Serialization.Metadata.JsonTypeInfo
+    public sealed partial class JsonTypeInfo<T> : System.Text.Json.Serialization.Metadata.JsonTypeInfo
```

Comment:
@stephentoub I'm making this change on the assumption that it isn't a breaking change. Is that correct?